adhocteam/Head-Start-TTADP
# Fix dashboard issues found in demo

## Description of change

There where a few issues found in the demo:

- Remove "Grants" from y-axis label of TTA Hrs chart.
- Fix Date Range warning message to be: "Reports are available from 09/01/2020. Use the format MM/DD/YYYY."
- Date Range selector prevents a Start Date of 09/01/2020.
- Dashboard Overview widget should show dollar sign icon '$' for Grants.


## How to test

Log into site the following should be fixed:

- Remove "Grants" from y-axis label of TTA Hrs chart.
- Fix Date Range warning message to be: "Reports are available from 09/01/2020. Use the format MM/DD/YYYY."
- Date Range selector prevents a Start Date of 09/01/2020.
- Dashboard Overview widget should show dollar sign icon '$' for Grants.


## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-258


## Checklists

### Every PR


~- [ ] API Documentation updated~
~- [ ] Boundary diagram updated~
~- [ ] Logical Data Model updated~
~- [ ] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions~

### Production Deploy

~- [ ] Staging smoke test completed~

### After merge/deploy
